src/main/java/com/playmonumenta/libraryofsouls/SoulHistoryEntry.java

```diff
@@ -105,7 +99,7 @@ public SoulHistoryEntry(NBTTagCompound nbt, long modifiedOn, String modifiedBy,
 		mHeight = height;
 
 		mName = GsonComponentSerializer.gson().deserialize(nbt.getString("CustomName"));
-		mLabel = Utils.getLabelFromName(PlainComponentSerializer.plain().serialize(mName));
+		mLabel = Utils.getLabelFromName(PlainTextComponentSerializer.plainText().serialize(mName));
```

Comment:
Not strictly required, but I'd be tempted to store `PlainTextComponentSerializer.plainText()` to a static/final variable so you don't need to have it typed out everywhere; it has no state to worry about, so reusing this is fine.

Reply:
Looks like this is already in the Utils class; I'd suggest using that.

Reply:
Good call, will do.